apps/mobile/src/screens/chat/chat.tsx

```diff
@@ -1,22 +1,22 @@
 import React from "react";
 
-import { Checkbox, ScreenWrapper, View } from "~/libs/components/components";
-import { useState } from "~/libs/hooks/hooks";
-import { globalStyles } from "~/libs/styles/styles";
+import { LoaderWrapper, ScreenWrapper } from "~/libs/components/components";
+import { DataStatus } from "~/libs/enums/enums";
+import { useAppSelector } from "~/libs/hooks/hooks";
+
+import { CategoriesForm } from "./components/categories-form";
```

Comment:
May be for consisitency it could be better to import from `./components/components` .